keps/sig-api-machinery/4006-transition-spdy-to-websockets/README.md

```diff
@@ -0,0 +1,1200 @@
+(e.g. pods per node) or new limits added by this KEP?
+
+Are there any tests that were run/should be run to understand performance characteristics better
+and validate the declared limits?
+-->
+
+```
+<<[UNRESOLVED]>>
+TBD - currently unimplemented
+<<[/UNRESOLVED]>>
+```
+
+### Troubleshooting
+
+<!--
+This section must be completed when targeting beta to a release.
+
+For GA, this section is required: approvers should be able to confirm the
+previous answers based on experience in the field.
+
+The Troubleshooting section currently serves the `Playbook` role. We may consider
+splitting it into a dedicated `Playbook` document (potentially with some monitoring
+details). For now, we leave it here.
+-->
+
+###### How does this feature react if the API server and/or etcd is unavailable?
+
+```
+<<[UNRESOLVED]>>
+TBD - currently unimplemented
+<<[/UNRESOLVED]>>
+```
+
+###### What are other known failure modes?
+
+<!--
+For each of them, fill in the following information by copying the below template:
+  - [Failure mode brief description]
+    - Detection: How can it be detected via metrics? Stated another way:
+      how can an operator troubleshoot without logging into a master or worker node?
+    - Mitigations: What can be done to stop the bleeding, especially for already
+      running user workloads?
+    - Diagnostics: What are the useful log messages and their required logging
+      levels that could help debug the issue?
+      Not required until feature graduated to beta.
+    - Testing: Are there any tests for failure mode? If not, describe why.
+-->
+
+```
+<<[UNRESOLVED]>>
+TBD - currently unimplemented
+<<[/UNRESOLVED]>>
+```
+
+###### What steps should be taken if SLOs are not being met to determine the problem?
+
+## Implementation History
+
+<!--
+Major milestones in the lifecycle of a KEP should be tracked in this section.
+Major milestones might include:
+- the `Summary` and `Motivation` sections being merged, signaling SIG acceptance
+- the `Proposal` section being merged, signaling agreement on a proposed design
+- the date implementation started
+- the first Kubernetes release where an initial version of the KEP was available
+- the version of Kubernetes where the KEP graduated to general availability
+- when the KEP was retired or superseded
+-->
+
+```
+<<[UNRESOLVED]>>
+TBD - currently unimplemented
+<<[/UNRESOLVED]>>
+```
+
+## Drawbacks
+
+<!--
+Why should this KEP _not_ be implemented?
+-->
+
+```
+<<[UNRESOLVED]>>
+TBD - currently unimplemented
+<<[/UNRESOLVED]>>
+```
+
+## Alternatives
+
+<!--
+What other approaches did you consider, and why did you rule them out? These do
+not need to be as detailed as the proposal, but should include enough
+information to express the idea and why it was not acceptable.
+-->
+
+```
+<<[UNRESOLVED]>>
+TBD - currently not finished
+<<[/UNRESOLVED]>>
+```
+
+<!--  DISCUSS why we're not using http/2.0. -->
```

Comment:
This is   http2 does not expose any streaming capabilities so we should have to implement again all the streaming protocols and maintain our low level http2 library

Reply:
Yes. I just put a placeholder to remind myself to flesh this out in a future PR.